Build-and-Deploy-Your-First-Dashboard.mdx

```diff
@@ -0,0 +1,420 @@
+  alt="Hero Dark"
+/>
+     </Frame>
+
+
+
+  </Accordion>
+    <Accordion title="4. Filter Preview">
+
+
+
+   <Frame>
+
+  <img
+  className="block dark:hidden"
+  src="/images/build-your-first-dashboard/light/example-dashboard-filter-dark.png"
+  alt="Hero Light"
+/>
+<img
+  className="hidden dark:block"
+  src="/images/build-your-first-dashboard/dark/example-dashboard-filter-dark.png"
+  alt="Hero Dark"
+/>
+     </Frame>
+
+
+
+  </Accordion>
+</AccordionGroup>
+
+<Tip>For more information on dashboard filters, check out our [Dashboard Filters Guide](/filters/filters-in-depth).</Tip>
+
+
+
+#### Add charts
+
+To add additional charts you can go to through the steps of creating a new chart. 
+
+Alternatively if you have already built a chart you can click on **Add Chart** to use on your dashboard.
+ <Frame>
+
+  <img
+  className="block dark:hidden"
+  src="/images/build-your-first-dashboard/light/add-chart-dark.png"
+  alt="Hero Light"
+/>
+<img
+  className="hidden dark:block"
+  src="/images/build-your-first-dashboard/dark/add-chart-dark.png"
+  alt="Hero Dark"
+/>
+     </Frame>
+
+
+
+
+## Share your dashboard
+
+You can share your dashboard via email with anyone in by clicking on the share button and sharing it as follows, if you would like a preview of the dashboard please click on the preview.
+
+Your team members will be invited to your dashboard. By sharing your dashboard they will become a team member and can collaborate on future dashboards with you.
+
+<Frame>
+<img
+  className="block dark:hidden"
+  src="/images/build-your-first-dashboard/light/share.png"
+  alt="Hero Light"
+/>
+<img
+  className="hidden dark:block"
+  src="/images/build-your-first-dashboard/dark/share-dark.png"
+  alt="Hero Dark"
+/>
+</Frame>
+
+
+<Note> If the invitee aceepts the invite, they will gain access to ALL dashboards and data in your organization!</Note>
+
+
+## User Authorization (Tenant Registration)
+
+To set up **tenant registration** please follow the steps outlined in [Backend Setup](deploy-dashboards/backend-setup). 
+
+Alternatively if you are using our **sample database** and do **not intend to go live** with the dashboard this step is optional.
+### Preview dashboard as your designated end users
+
+If you have completed the [Backend Setup](deploy-dashboards/backend-setup), you are able to view the dashboard as one of your customers.
+
+If you are using the **sample database** we have provided two sample tenants, which you are able to preview in the hub below. 
+
+ <Frame>
+
+  <img
+  className="block dark:hidden"
+  src="/images/build-your-first-dashboard/light/preview-as-dark.png"
+  alt="Hero Light"
+/>
+<img
+  className="hidden dark:block"
+  src="/images/build-your-first-dashboard/dark/preview-as-dark.png"
+  alt="Hero Dark"
+/>
+     </Frame>
+
+     If you are using the **sample database** and want to add more to add more **Preview Tenants**, please click on **Home**, then **Deploy** and then **Registered Tenants**
```

Comment:
Rewrite to:

If you are using the sample database, the below is the steps and an example of how to create a new testing tenants:

Reply:
also include a link out to the tenancy management page, and ideally a section in that page that explains how tenancy and data permissioning work based on tenancy